docs/docs/about/modelling/setup/models/compressor_modelling/fixed_speed_pressure_control/index.md

```diff
@@ -6,11 +6,45 @@ description: Introduction to fixed speed pressure control
 
 # Fixed speed pressure control
 
-Generally, the work performed by a compressor train can be reduced/increased by changing the rotational speed
-of the compressor train shaft. In some cases there is a need to reduce the discharge pressure from a compressor
-train without changing the rotational speed of the shaft. Examples can be:
+## Theory
 
-- The compressor train only operates at one speed (a [SINGLE_SPEED_COMPRESSOR_TRAIN](../compressor_models_types/single_speed_compressor_train_model.md)),
+Compressors are typically controlled by changing the rotational speed of the compressor train shaft, which can either increase or decrease the work performed. 
+In the case where this is not possible ([single speed compressors](../compressor_models_types/single_speed_compressor_train_model.md)), or this is not sufficient to control the compressor, other methods of control need to be used.
+In eCalc™, there are three main compressor control methods considered (aside from rotational speed control), these are:
+
+- Upstream choking/throttling
+- Downstream choking/throttling
+- Anti-surge recycling
+
+Each of these methods are used depending on the situation and placement of the compressor's operating points. 
+A summary of how these methods work in practice are seen below. 
+In these figures, the red point represents a singular operating point, which is shifted with the given control mechanism. In each case the operating points starts in the below the compressor curve and it then adjusted (as head and rate is manipulated) along it as it is further controlled. 
```

Comment:
**In each case the operating points starts in the below the compressor curve and it then adjusted (as head and rate is manipulated) along it as it is further controlled.** 

This sentence is a bit funny. Seems like there are some words that should be added or removed.

Reply:
Yip! I agree, I have re-worded it